src/word_wrap/Wrapper.java

```diff
@@ -3,9 +3,9 @@
 public class Wrapper {
 
     public static String wrap(String s, int column) {
-        if (s.length() < column) {
+        if (s.length() <= column) {
             return s;
         }
-        return s.substring(0, column) + "\n" + s.substring(column, s.length());
+        return s.substring(0, column) + "\n" + wrap(s.substring(column, s.length()), column);
```

Comment:
Just read it. And your blog posts. Great reads. So funny that Uncle Bob's Post is all about Word Wrap too!